README.md

```diff
@@ -1,109 +1,50 @@
 # MN Bootstrap
 
-### Pre-requisites to be Installed
+> Distribution package for Dash Evolution Masternode installation
 
-* [Python](https://www.python.org/downloads/)
-* [docker](https://docs.docker.com/engine/installation/) (version 17.04.0+)
-* docker-compose (`pip install -U docker-compose`)
+## Table of Contents
 
-### Setup
+- [Pre-requisites](#Pre-requisites)
+- [Install](#install)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
 
-0. Clone this repo & cd to the directory:
+## Pre-requisites
 
-```
-git clone git@github.com:dashevo/mn-bootstrap.git ./mn-bootstrap
-cd mn-bootstrap
-```
-
-### Using mn-bootstrap.sh for regtest
+* [Docker](https://docs.docker.com/engine/installation/)
+* [Docker compose](https://docs.docker.com/compose/install/)
 
-mn-bootstrap provides a wrapper around docker-compose to make using different networks
-and presets easier. It is called this way:
+## Install
 
-```bash
-$ ./mn-bootstrap.sh <network> [preset] <compose_args...>
-```
-
-To bootstrap with regtest as network and latest preset use:
-
-```bash
-$ ./mn-bootstrap.sh regtest up -d
-```
+Download and unzip [package](https://github.com/dashevo/mn-bootstrap/archive/master.zip).
 
-To bootstrap with regtest as network and specific preset use:
+## Usage
 
-```bash
-$ ./mn-bootstrap.sh regtest maithai up -d
-```
+Package contains Docker Compose file and configuration presets.
 
-The argument `-d` is is used to start everything in the background. User `logs`
-to view the logs in the foreground:
+### Configure
 
-```bash
-$ ./mn-bootstrap.sh regtest logs
-```
+Package contains several configuration presets:
+ - Local - standalone masternode for local development
+ - Evonet - masternode with Evonet configuration
 
-If you want to call dash-cli inside of the dashd container, you can use the `dash-cli.sh` wrapper
+There are two ways to apply a present:
+ 1. Rename corresponding dotenv file (i.e. `.env.local`) to `.env`
+ 2. Add `--env-file` option to `docker-compose` command (requires `docker-compose` v1.25.0+)
```

Comment:
@thephez Does it make sense to specify minimal versions in "Pre-requisites" section?

Reply:
Yes, good idea. I overlooked that. :+1: